src/components/common/formFields/Widgets/Widgets.tsx

```diff
@@ -88,6 +109,24 @@ export const StyledInput = (props: StyledInputPropsType): JSX.Element => {
 }
 
 export const StyledTextarea = (props: StyledInputPropsType): JSX.Element => {
+    const [inputValue, setInputValue] = useState(props.value || '')
```

Comment:
This is repetitive code.

Reply:
Can we try to avoid duplicacy?

Reply:
Done... Initially, it was built a little differently.